app/helpers/application_helper.rb

```diff
@@ -1,2 +1,24 @@
 module ApplicationHelper
-end
+  def find_type(url)
+    api_key = Rails.application.secrets.embedly_api_key
+    embedly_api = Embedly::API.new(key: api_key)
+    obj = embedly_api.oembed :url => url
+    obj.first.type
+  end
+  def embed_media(url, size)
```

Comment:
Put an empty line between each method definition so its easier to scan and read this file.